template-parts/highlightedSection.php

```diff
@@ -0,0 +1,21 @@
+<?php
+ $enable = get_field('enable_highlighted_section_cf');
+ $title = get_field('highlighted_section_title');
+ $content = get_field('highlighted_section_content_section');
+ $link = get_field('highlighted_section_link_to_blog_post');
+ if($enable && $title && $content && $link): ?>
```

Comment:
It's best to separate this check for each component, so that even if I only have content without a title, the section still appears.